packages/core/src/types.ts

```diff
@@ -33,6 +33,15 @@ type TextFilterAndroid = {
   replace: string;
 };
 
+type CommonStyleKeys = Extract<keyof CSSProperties, keyof ViewStyle>;
+
+/**
+ * Unified style type for web and native
+ */
+type UnifiedViewStyle = {
+  [K in CommonStyleKeys]: CSSProperties[K] & ViewStyle[K];
+};
+
```

Comment:
I don't think it's a good idea to have a unified style since css properties will not be applied to native code.

Reply:
Usually, people would think that if they apply those CSS properties, they should be applied to native apps as well. That can create confusion.